### PR TITLE
Task1 done

### PR DESCRIPTION
**Pull request by snezhanabozkurt**

### CODE REVIEW

**Comment by HsynHsyn** [comment]
goog job

File: src/snezhana/Task1.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        //Write a method that can identify  a given number is even or odd.
+        int number = 5;
+        
+        if (number % 2 == 0) {
+            System.out.println(number + " is even");
+        } else {
+            System.out.println(number + " is odd");
+        }
+    }
+
```
Comment (HsynHsyn) [inline src/snezhana/Task1.java:23]: //good job

**Comment by snezhanabozkurt** [comment]
Job done

**Comment by Homeless123** [comment]
it perfect

**Pull request closed without merging**